packages/react-data-grid/src/HeaderCell.tsx

```diff
@@ -99,12 +99,12 @@ export default class HeaderCell extends React.Component<Props> {
       </div>
     );
 
-    if (rowType === HeaderRowType.HEADER && column.draggable) {
-      const DraggableHeaderCell = this.props.draggableHeaderCell;
+    const DraggableHeaderCell = this.props.draggableHeaderCell;
+    if (rowType === HeaderRowType.HEADER && column.draggable && DraggableHeaderCell) {
       return (
         <DraggableHeaderCell
           column={column}
-          onHeaderDrop={this.props.onHeaderDrop}
+          onHeaderDrop={this.props.onHeaderDrop!}
```

Comment:
We need a better way to validate `onHeaderDrop` prop. May be we can make `onHeaderProp` optional in `DraggableHeaderCell` but drag/drop does not work without it. I am not a fan of using a wrapper for adding draggable support 

https://github.com/adazzle/react-data-grid/blob/next/examples/scripts/example24-draggable-header.js#L80